styles/helpers/flex-grid.css

```diff
@@ -0,0 +1,5 @@
+.flex {
```

Comment:
Since this wasn't dependant on size nor layout I thought that's the only thing that made sense to leave out of the mixin. However we can add it for consistency's sake.